src/AspNet.Security.OAuth.Discord/DiscordAuthenticationOptions.cs

```diff
@@ -28,6 +27,14 @@ public class DiscordAuthenticationOptions : OAuthOptions
         /// </summary>
         public string DiscordAvatarFormat { get; set; } = Urls.AvatarUrlFormat;
 
+        /// <summary>
+        /// Controls how the authorization flow handles existing authorizations.
+        /// The default value of this property is null and the "prompt" query string parameter will not be appended to the Authorization Endpoint URL.
+        /// Assigning this property any value other than null, empty, or whitespace will automatically append the "prompt" query string parameter to the Authorization Endpoint URL,
+        /// with the corresponding value.
+        /// </summary>
+        public string? DiscordAuthenticationPrompt { get; set; }
```

Comment:
I suggest we name it `Prompt` to simplify things (it's sadly too late for the other properties 😭)

Reply:
Okay, sounds good.  I'll work to make these changes.  Thanks